gemsfx/src/main/java/com/dlsc/gemsfx/util/UIUtil.java

```diff
@@ -209,7 +208,27 @@ public static void copyToClipboard(String copyContent) {
      * @throws NullPointerException if the event is null.
      */
     public static boolean clickOnNode(MouseEvent event) {
-        return event.getButton() == MouseButton.PRIMARY && event.getClickCount() == 1 && event.isStillSincePress();
+        return clickOnNode(event, false);
+    }
+
+    /**
+     * Determines if the given mouse event is a primary button click
+     * that hasn't moved since it was pressed.
+     *
+     * <p>This method checks if the mouse event satisfies the following conditions:
+     * <ul>
+     *   <li>The mouse button is the primary button (usually the left button).</li>
+     *   <li>The mouse has not moved since it was pressed.</li>
+     *   <li>If {@code isSingleClick} is {@code true}, the event must be a single click.</li>
+     * </ul>
+     *
+     * @param event        The mouse event to check. Must not be null.
+     * @param isSingleClick {@code true} if the event must be a single click, {@code false} otherwise.
+     * @return {@code true} if the event is a stable primary button click, {@code false} otherwise.
+     * @throws NullPointerException if the event is null.
+     */
+    public static boolean clickOnNode(MouseEvent event, boolean isSingleClick) {
```

Comment:
"isStillSincePressed" can never be "still". "still" means that the mouse has not been moved between the last two pressed events.

Reply:
Thank you for your guidance. After testing, it turned out to be exactly as you said. If it is a mouse press or release, the first triggered event always returns false for isStillSincePress.